README.md

```diff
@@ -1 +1,25 @@
-# redux-debounce-field
\ No newline at end of file
+# redux-debounce-field
+A variant of redux-form's `<Field>` component, but with a debounce feature that delay's the field's `onChange` event from triggering.
+
+## Usage
+```js
+import DebounceField from 'redux-debounce-field';
+
+const MyInput = ({
+  input,
+  meta,
+  ...rest
+}) => (
+  <input
+    {...input}
+    {...rest}
+  />
+);
+
+const MyForm = () => (
+  <DebounceField
+    wait={250}
```

Comment:
Does this have a default? If it does we should document the props to this component and what their default are.

Reply:
yes it does have a default, I've updated the docs